src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -781,6 +782,90 @@ def test_containerapp_dev_service_binding_e2e(self, resource_group):
 
         self.cmd(f'containerapp env delete -g {resource_group} -n {env_name} --yes')
 
+    @live_only()
```

Comment:
May I ask why this test is marked as live only?

Reply:
same thing from above. both're non-live now!!